public_officials/templates/public-officials/user_show.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script type="text/javascript" src="/static/public_officials/bootstrap-3.3.7-dist/js/bootstrap.min.js"></script>
+    <link href="/static/public_officials/bootstrap-3.3.7-dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="/static/public_officials/stylesheets/main.css" rel="stylesheet">
+    <title>Will Legislate For Money</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-default">
+      <div class="container-fluid">
+        <div class="navbar-header">
+          <a class="navbar-brand" href="{% url 'home' %}">Home</a>
+        </div>
+
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+          <ul class="nav navbar-nav navbar-right">
+              <li><a href="{% url 'bills_index' %}">Recent Bills</a></li>
+              <li><a href="{% url 'senator_index' %}">Senators</a></li>
+              <li><a href="{% url 'representative_index' %}">Representatives</a></li>
+              <li><a href="{% url 'about' %}">About</a></li>
+              {% if request.user.is_anonymous %}
+              <li><a href="{% url 'social:begin' 'twitter' %}">Login with Twitter</a></li>
```

Comment:
We'll have to research and see if there's a way to make partials. Seems simple, but probably isn't.